react/finger-exercises/ecmascript/01 - spread/index.js

```diff
@@ -1,10 +1,21 @@
 import { isArray } from './utils';
 
-export function min() {
-  isArray();
-  return Math.min();
+export function min(...args) {
+  if (args.length) {
+    if (args.length > 1) return Math.min(...args);
+    if (isArray(args[0])) return Math.min(...args[0]);
+    return Math.min(args[0]);
+  }
+  return undefined;
 }
 
-export function copy() {
+export const copy = param => Object.assign(param.constructor(), param);
```

Comment:
In this case we should use de spread operator and make sure `copy` works for `Object` and `Array` as well.